### PR TITLE
Add quickstart guide for rx65nxcloudkit development and associated as…

### PR DESCRIPTION
**Pull request by nmatthews-witekio**
…sets

This is really a test branch to see how easy it is to create the quickstart guide in github markdown.

Need to think a bit about how to handle assets, etc. -- do we include original screenshots as well as scaled ones, etc.

### CODE REVIEW

**Comment by nmatthews-witekio** [comment]
Need to review all assets, as well as the markdown.

**Comment by nik-markovic** [comment]
> Need to review all assets, as well as the markdown.

PNG doesn't compress well with gradients and non-single-color backgrounds. If you really need to capture the full desktop, when I'd suggest removing the background flower. You can use **ALT+PrintScreeen** to capture only one window or a modal dialog without the unneccessary scenery.

You shouldn't screenshot every step along the way - only those which may be hard to follow or where if there's value in the actual screenshot. Like, do we actually need to screenshot something that can be described with a step "Click the *Next* button" or "accept the license agreement" for example?

The thing takes forever to load. git lone seemed to take noticably long.

The md file you commited should be called QUICKSTART_DEMO.md or QUICKSTART_GUIDE.md (in upper case) if more appropriate, like the test of the quickstart demos in the repo.

The branch name has underscores. We use dasahes for all branches.

The image file extensions will work, but having them retain original extension (png/jpg) would be nice for purpose of previews and such.
